.github/workflows/self-push.yml

```diff
@@ -263,13 +263,6 @@ jobs:
         run: |
           nvidia-smi
 
-      - name: Install dependencies
-        run: |
-          apt -y update && apt install -y libaio-dev
-          pip install --upgrade pip
-          rm -rf ~/.cache/torch_extensions/ # shared between conflicting builds
-          pip install .[testing,deepspeed,fairscale]
-
```

Comment:
No need this step -> installed in the image `huggingface/transformers-pytorch-deepspeed-latest-gpu`, except `fairscale` (which we don't test at this moment)